tests/test_bot.py

```diff
@@ -363,6 +364,56 @@ def test_send_reply(self):
                                 attachments='null')
 
 
+class TestSendMessageWithMessageAttachments(TestPhialBot):
+    '''Test phial's send_message function with message attachments'''
+
+    def test_send_message(self):
+        self.bot.slack_client = MagicMock()
+        self.bot.slack_client.api_call = MagicMock(return_value="test")
```

Comment:
Does this have to have a `return_value`? It doesn't look like you are checking it anywhere